docs/guidance/EnterpriseScaleAnalytics-NetworkArchitecture.md

```diff
@@ -1,6 +1,6 @@
 # Network Architecture Considerations
 
-Enterprise-Scale Analytics promises the possibility to easily share and access datasets across multiple data domains and Data Landing Zones without critical bandwidth limitations and without creating multiple copies of the same dataset. To deliver on that promise, different network designs had to be considered, evaluated and tested to make sure that these are compatible with the existing Hub and Spoke and vWAN deployments of corporations. Based on the current capabilities of Azure Networking Services it is recommended to rely on a meshed network architecture. What this means is that it is recommended to setup Vnet peering between
+Data Management & Analytics Scenario promises the possibility to easily share and access datasets across multiple data domains and Data Landing Zones without critical bandwidth limitations and without creating multiple copies of the same dataset. To deliver on that promise, different network designs had to be considered, evaluated and tested to make sure that these are compatible with the existing Hub and Spoke and vWAN deployments of corporations. Based on the current capabilities of Azure Networking Services it is recommended to rely on a meshed network architecture. What this means is that it is recommended to setup Vnet peering between
```

Comment:
Reframe this paragraph? Consider ... 

"The network design goals for Data Management & Analytics architecture are to enable network path between Data Landing Zones which minimizes latency; and maximises bandwidth. To meet these goals, several options were evaluated; considered and tested with traditional hub and spoke and vWAN topologies. Based on the current capabilities of Azure Networking services, it is recommended to use a mesh network architecture. "

Reply:
I think in this article, we need to call-out that the recommendations are applicable to both vWAN and traditional hub-spoke design. This was a feedback from a customer who thought that the points are only applicable to hub-spoke since all the images; diagrams show hub-spoke and none show vWAN.